packages/api-explorer/src/lib/markdown/renderer.js

```diff
@@ -78,13 +77,12 @@ renderer.link = function link(href, title, text) {
 
   const blog = href.match(/^blog:([-_a-zA-Z0-9#]*)$/);
   if (blog) {
-    uiSref = `blog.show({'blog': '${blog[1]}'})`;
-    href = '';
+    href = `/blog/${blog[1]}`;
```

Comment:
I know you put in the issue to change it to '/blog/:blog', but how will it have access to the variable blog shouldn't we being use string interpolation?

Reply:
This line of code looks correct to me?